airbyte-integrations/connectors/source-mention/manifest.yaml

```diff
@@ -0,0 +1,11358 @@
+          path: >-
+            /accounts/{{ stream_partition.account_id }}/alerts/{{
+            stream_partition.alert_id }}/mentions/{{ stream_partition.mention_id
+            }}
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - mention
+              - tasks
+        partition_router:
+          - type: SubstreamPartitionRouter
+            parent_stream_configs:
+              - type: ParentStreamConfig
+                parent_key: id
+                partition_field: alert_id
+                stream:
+                  $ref: "#/definitions/streams/Alerts"
+          - type: SubstreamPartitionRouter
+            parent_stream_configs:
+              - type: ParentStreamConfig
+                parent_key: id
+                partition_field: mention_id
+                stream:
+                  $ref: "#/definitions/streams/Mentions"
+          - type: SubstreamPartitionRouter
+            parent_stream_configs:
+              - type: ParentStreamConfig
+                parent_key: id
+                partition_field: account_id
+                stream:
+                  $ref: "#/definitions/streams/Accounts"
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/Tasks"
+  base_requester:
+    type: HttpRequester
+    url_base: https://api.mention.net/api
+    authenticator:
+      type: ApiKeyAuthenticator
+      api_token: "{{ config[\"api_key\"] }}"
+      inject_into:
+        type: RequestOption
+        field_name: Authorization
+        inject_into: header
+
+streams:
+  - $ref: "#/definitions/streams/Alerts"
+  - $ref: "#/definitions/streams/Alert"
+  - $ref: "#/definitions/streams/Mentions"
+  - $ref: "#/definitions/streams/Mention"
+  - $ref: "#/definitions/streams/Mention Children"
+  - $ref: "#/definitions/streams/Accounts"
+  - $ref: "#/definitions/streams/Account"
+  - $ref: "#/definitions/streams/Tags"
+  - $ref: "#/definitions/streams/Alert Authors"
+  - $ref: "#/definitions/streams/Alert Tasks"
+  - $ref: "#/definitions/streams/Statistics"
+  - $ref: "#/definitions/streams/Tasks"
+
+spec:
+  type: Spec
+  connection_specification:
+    type: object
+    $schema: http://json-schema.org/draft-07/schema#
+    required:
+      - api_key
+    properties:
+      api_key:
+        type: string
+        order: 0
+        title: API Key
+        airbyte_secret: true
+      from:
+        type: string
+        description: stats are to be fetched for mentions retrieved after that date
+        order: 1
+        title: from
+      to:
+        type: string
+        description: stats are fetched for mentions retrieved before that date
+        order: 2
+        title: to
+      interval:
+        type: string
+        description: >-
+          Periodicity of statistics returned. it may be daily(P1D), weekly(P1W)
```

Comment:
It looks from and to are required to get stats. Can you raise a discussion in connector-dev Slack channel about what is the best implementation in this case?

Reply:
Okay, I've sent a message there!